library/data/geofoncier_wms_getFeatureInfo/geofoncier.js

```diff
@@ -0,0 +1,39 @@
+lizMap.events.on({	
+	'lizmappopupdisplayed': function(e) {	
+		//get table parent if h4 = Dossiers des Géomètres Experts
+		let popupGE = $('#popupcontent .lizmapPopupContent').find('h4:contains("Dossiers des Géomètres Experts")').parent();
```

Comment:
Pour configure le script plus facilement, peut être ajouter une variable ` expectedParentTile = "Dossiers ..." `

puis faire le find en utilisant cette variable ?

Reply:
C'est noté merci. Je ferai une V2 du code.
Aux admin du dépôt pourriez-vous svp merger le commit ?

Merci d'avance.